app/src/main/java/org/hackillinois/android/model/Event.java

```diff
@@ -0,0 +1,19 @@
+package org.hackillinois.android.model;
+
+public class Event {
+    private String name;
+    private String description;
+    private long startTime;
+    private long endTime;
+
+    public Event(String name, String description, long startTime, long endTime) {
```

Comment:
This can be shortened a lot by using a Kotlin data class:

```Kotlin
package org.hackillinois.android.model

data class Event(val name: String, val description: String, val startTime: Long, val endTime: Long)
```

This will automatically create getters for each constructor parameter, as well as `hashCode()` and `equals()` functions based on them. If there's no need for those two functions, we can remove the `data` modifier and it should function the same.

Reply:
You're a wizard.